fbw-a380x/src/systems/systems-host/systems/LegacyFuel.ts

```diff
@@ -127,6 +135,11 @@ export class LegacyFuel implements Instrument {
   }
 
   public onUpdate(): void {
+    const dt = this.sysHost.deltaTime;
+    if (!this.hasInit || this.throttler.canUpdate(dt) <= 0) {
+      return;
+    }
```

Comment:
TODO FIXME: Use ClockEvents instead of deprecated UpdateThrottler

Reply:
I am more with @tracernz on this one, adding some sort of deltatime or update rate to the actual backplane instrument cause doing this in each on yourself seems counterintuitive to the original idea

Reply:
I agree, maybe let's just put the FIXME inside index.ts of the systemshost to have it noted down somewhere.

Reply:
Yes, I also wouldn't know why UpdateThrottler itself is deprecated